compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -3236,32 +3236,42 @@ OMR::Power::CodeGenerator::fixedLoadLabelAddressIntoReg(
       bool useADDISFor32Bit)
    {
    TR::Compilation *comp = self()->comp();
```

Comment:
looks good. maybe the line of getting comp be eliminated, since local comp is not used at all.

Reply:
Fixed